README.md

```diff
@@ -16,9 +16,11 @@ Installing the Package
 ======================
 
 Here is a FireboltNetSDK [NuGet page](https://www.nuget.org/packages/FireboltNetSDK/).
+Currently  installed version is 1.1.0. The preferable way to update version is using release GitHub action, however this can be done manually as well using CLI. 
```

Comment:
I'm not sure what it's about. Users can't install the .net sdk using a github action. Also not sure what `Currently  installed version is 1.1.0` means, please explain

Reply:
removed